tests/rip7560/rip7560TestUtils.go

```diff
@@ -0,0 +1,122 @@
+package rip7560
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/status-im/keycard-go/hexutils"
+	"math/big"
+	"os"
+	"testing"
+)
+
+const PREDEPLOYED_SENDER = "0xed0a7aabc745fe5d8bc6ad625c767df86044d049"
+
+type testContext struct {
+	genesisAlloc types.GenesisAlloc
+	t            *testing.T
+	chainContext *ethapi.ChainContext
+	config       *params.ChainConfig
+	gaspool      *core.GasPool
```

Comment:
I don't think `gaspool` belongs to a `testContext` as it is pretty likely you will want a test with, for example, two transactions. I think `gaspool` is designed to track a single transaction's gas.